include/deal.II/distributed/grid_tools.h

```diff
@@ -134,6 +134,60 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   const std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> &pack,
                                   const std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> &unpack);
+
+    /**
+     * A structure that allows the transfer of data of type T from one processor
+     * to another. It corresponds to a packed buffer that stores a list of
+     * cells and an array of type T.
```

Comment:
This makes it sound like we use different containers for the cells and for the `T`s, but we really just use `std::vector` for both. Please make sure that you use the same noun for both.